server/src-lib/Hasura/RQL/DML/Select/Types.hs

```diff
@@ -243,21 +243,22 @@ traverseAnnField f = \case
   AFNodeId qt pKeys -> pure $ AFNodeId qt pKeys
   AFExpression t -> AFExpression <$> pure t
 
-type AnnField = AnnFieldG S.SQLExp
+type AnnField b = AnnFieldG b S.SQLExp
 
-data SelectArgsG v
+data SelectArgsG (b :: Backend) v
   = SelectArgs
-  { _saWhere    :: !(Maybe (AnnBoolExp v))
-  , _saOrderBy  :: !(Maybe (NE.NonEmpty (AnnOrderByItemG v)))
+  { _saWhere    :: !(Maybe (AnnBoolExp b v))
+  , _saOrderBy  :: !(Maybe (NE.NonEmpty (AnnOrderByItemG b v)))
   , _saLimit    :: !(Maybe Int)
   , _saOffset   :: !(Maybe S.SQLExp)
```

Comment:
TODO: Phase 2: `S.SQLExp` -> `SQLExp b` and `PGCol` too

Reply:
PGCol done.